src/sparseml/pytorch/torchvision/train.py

```diff
@@ -25,9 +25,14 @@
 from torch.utils.data.dataloader import default_collate
 from torchvision.transforms.functional import InterpolationMode
 
+from sparseml.pytorch.image_classification.utils.helpers import (
+    _download_model_from_zoo_using_recipe,
```

Comment:
could we make this imported function non private?

Reply:
Oh wait we should be using a non image_classification helper for this, good catch

Reply:
Updated to use non-private, thanks!